# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../clockspeed/makelib.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./make-makelib.sh.orig	2008-03-10 23:58:08.000000000 +0000
+++ ./make-makelib.sh	2008-03-10 23:58:46.000000000 +0000
@@ -1,6 +1,6 @@
 echo 'main="$1"; shift'
 echo 'rm -f "$main"'
-echo 'ar cr "$main" ${1+"$@"}'
+echo '${AR:-ar} cr "$main" ${1+"$@"}'
 
 case "$1" in
 sunos-5.*) ;;
@@ -11,6 +11,6 @@
 hp-ux-*) ;;
 sco*) ;;
 *)
-  echo 'ranlib "$main"'
+  echo '${RANLIB:-ranlib} "$main"'
   ;;
 esac
